string attacker_name
string target_name
float32 damage_dealt
bool is_kill
geometry_msgs/Point location
